src/seedu/addressbook/commands/SortCommand.java

```diff
@@ -0,0 +1,45 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.person.ReadOnlyPerson;
+import seedu.addressbook.data.person.UniquePersonList;
+
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+
+public class SortCommand extends Command{
+
+    public static final String COMMAND_WORD = "sort";
+
+    //To change
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Displays all persons in the address book as a sorted list with index numbers.\n"
+            + "Example: " + COMMAND_WORD;
+
+
+   public CommandResult execute() throws UniquePersonList.DuplicatePersonException {
```

Comment:
Incorrect indentation? Basic indentation should be 4 spaces according to coding standard